include/macros.h

```diff
@@ -64,6 +64,9 @@
 
 #define HIGH_SCORE(score) (gSaveContext.highScores[score])
 
+#define GET_ITEMGETINF(flag) (gSaveContext.itemGetInf[flag >> 4] & (1 << (flag & 0xF)))
+#define SET_ITEMGETINF(flag) (gSaveContext.itemGetInf[flag >> 4] |= (1 << (flag & 0xF)))
```

Comment:
No macro to unset?
```c
#define UNSET_ITEMGETINF(flag) (gSaveContext.itemGetInf[flag >> 4] &= ~(1 << (flag & 0xF)))
```

Reply:
I think we would call that "clear"?

The game does clear (few) flags from the other three arrays, but not flags from `itemGetInf`
This may have to do with the purpose of `itemGetInf` to be permanent maybe? since it's for items supposedly

So on one hand it would be more consistent with the other arrays to have a clear macro,
and on the other hand `itemGetInf` flags are probably not meant to be cleared ever